course02/project01/gavin/Asteroids/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Its usually good to use the oldest possible version here, and not just your version. On my machine I have a slightly older cmake (3.10) and it doesnt work due to this line, although you are not using any new features from 3.13.